packages/logo-grid/style.css

```diff
@@ -1,11 +1,12 @@
-@import '~tippy.js/themes/light.css';
+@import '~@tippyjs/react/node_modules/tippy.js/dist/tippy.css';
```

Comment:
`tippy.js` is not getting hoisted now, so need to reference it as nested.

Reply:
I'm worried about this - if it is hoisted in a host installation doesn't this break?

Reply:
Yeah I thought about that as well...let me do some quick validation here.

Reply:
So far this has been working just fine. As we are no longer depending directly on `tippy.js`, it should always be nested in `@tippyjs/react`. Tested on www with npm v6 and v7.

I agree it's not ideal, but unfortunately I can't think of a better alternative.

Reply:
Yeah I can't really either. The issue I anticipate is if a project uses two different packages that both have tippy as a dep and it decides to hoist them. I guess we can just deal with this if/when it happens. Setting the version ranges using a carat seems to help to prevent things from being hoisted in this way a little.